_posts/2020-10-04-newpipe-0.20.0.md

```diff
@@ -1,154 +1,176 @@
+[@TobiGr](https://github.com/TobiGr) brought two more improvements: The setting to remember pop-up size and position has a less redundant description, and when switching to another app, the main player now minimizes to play audio in the background by default. Seamless transition. Swoosh.
 
-It's 2020, and our default resolutions also realized this. Instead of having 360p everywhere, main player videos are opened in 720p at 60FPS now, and pop-up ones in 480p. Thank you, [@B0pol](https://github.com/B0pol). NewPipe now also remembers what kind of media you downloaded the last time, and will auto-select it when you hit download on a new video. This was done by [@vmazoyer](https://github.com/vmazoyer).
+It is the current year, 2020, in higher resolution. Instead of having 360p everywhere, main player videos are opened in 720p at 60FPS, and pop-up ones in 480p. Thank you, [@B0pol](https://github.com/B0pol). NewPipe now also remembers what kind of media you downloaded last time, and will auto-select it when you hit "Download" on a new video. This was done by [@vmazoyer](https://github.com/vmazoyer).
 
-Since 'Autoplay' is a new configurable setting in this release, [@opusforlife2](https://github.com/opusforlife2) changed the 'Autoplay' toggle in the video details to say "Auto-queue" instead. Now it's clearer what that toggle does without leaving room for any confusion.
+Since 'Autoplay' is a new configurable setting in this release, [@opusforlife2](https://github.com/opusforlife2) changed the 'Autoplay' toggle in the video details to say "Auto-queue" instead. Now it's clearer what that toggle does without leaving room for confusion.
 
 ### More Translation updates and improvements
 
-Apart from the language improvements mentioned above, some other interesting localization changes happened in this release. [@comradekingu](https://github.com/comradekingu), a frequent contributor to and language checker of NewPipe and other open source projects, has once again taken a close look at our strings. There was, as always, a lot of room for improvement. Our wonderful translators rushed to help and immediately implemented his suggestions.
+On the topic of language, you may find it improved in this release, and in other forms of written NewPipe material. [Allan Nordhøy](https://github.com/comradekingu) landed another pleasant and hopefully indistingishable effort. There was, as always, unmaintained walls leaning on room for improvement. The always wonderful translators have carried those improvements onto all languages and some in turn back into English.
 
-We are also happy to provide **new localizations** in this release: Sardinian, Bengali, Portuguese (Portugal), Neapolitan and Berber.
```

Comment:
Who is "we". Not sure "Berber" is a language? Isn't it Berber something?

Reply:
The team that makes the app. Team NewPipe.